hrm-service/README.md

```diff
@@ -12,6 +12,7 @@ RUNNING_TEST=true
 Then, in the `hrm-service` directory, run
 
 ```shell
+docker compose -f ./docker-compose-test.yml build
```

Comment:
Maybe leave a small comment indicating that this is needed once (until docker config or db dump changes)

Reply:
@GPaoloni comment has been added